src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -104,47 +118,126 @@
            :separator                         [rn/view {:width 12}]
            :data                              communities
            :render-fn                         render-fn
-           :render-data                       {:featured? true
-                                               :width     @view-size
+           :render-data                       {:width     @view-size
                                                :view-type view-type}}])])))
 
+(defn discover-communities-header
+  [{:keys [featured-communities-count
+           featured-communities
+           view-type
+           selected-tab]}]
+  [react/animated-view
+   [screen-title]
+   [featured-communities-header featured-communities-count]
+   [featured-list featured-communities view-type]
+   [quo/separator]
+   [discover-communities-segments selected-tab]])
+
 (defn other-communities-list
-  [communities view-type]
-  [rn/flat-list
-   {:key-fn                            :id
-    :keyboard-should-persist-taps      :always
-    :shows-horizontal-scroll-indicator false
-    :separator                         [rn/view {:margin-bottom 16}]
-    :data                              communities
-    :render-fn                         render-fn
-    :render-data                       {:featured? false
-                                        :width     "100%"
-                                        :view-type view-type}}])
+  [{:keys [communities communities-ids view-type]}]
+  [rn/view {:flex 1}
+   (map-indexed
+    (fn [inner-index item]
+      (let [community-id (when communities-ids item)
+            community    (if communities
+                           item
+                           [rf/sub [:communities/home-item community-id]])]
+        [rn/view
+         {:key           (str inner-index (:name community))
```

Comment:
Any reason why we're not using `(:id community)` as the key?

Reply:
no specific reason.